### ISSUE TITLE
Accessibility - No heading levels can be set for the highcharts title

### ISSUE THREAD

**Issue by priyassilect** [highcharts/highcharts#16984]
#### Expected behaviour
The chart titles should display heading levels when tools like Accessibility Insights for Web is used.

#### Actual behaviour
The chart titles do not show any heading levels when tools like Accessibility Insights for Web is used.

#### Live demo with steps to reproduce
Create any chart with a title. Use the addon (https://accessibilityinsights.io/docs/en/web/overview/). Turn on Headings through the extension. The titles in the page should be marked with respective heading levels including the chart title.
Enabled the Accessibility Module . Tried to use useHTML for the title with appropriate Heading levels and aria attributes. But the title will not be marked as a heading by the AI tool.

#### Product version
Highcharts v9 3.2

#### Affected browser(s)
Edge, Chrome

**Comment by oysteinmoseng** [comment]
Hi, thank you for reporting.

This is a planned enhancement, but tricky to implement due to reading order. For now the heading is marked as the appropriate heading level in an offscreen HTML region before the chart. The heading level is automatically computed from other headings on the page, but can be customized with the [accessibility.screenReaderSection.beforeChartFormat](https://api.highcharts.com/highcharts/accessibility.screenReaderSection.beforeChartFormat) option.

Example: https://jsfiddle.net/oysteinmoseng/cksh7t9e/show

<img width="717" alt="Screen Shot 2022-02-11 at 12 26 56" src="https://user-images.githubusercontent.com/5276642/153583766-92957d4b-3291-4065-928d-1381046a3271.png">
